services/network/network.go

```diff
@@ -684,7 +684,7 @@ type InterfaceIPConfigurationPropertiesFormat struct {
 	// Primary - Gets whether this is a primary customer address on the network interface.
 	Primary *bool `json:"primary,omitempty"`
 	// PublicIPAddress - Public IP address bound to the IP configuration.
-	PublicIPAddress *PublicIPAddress `json:"publicIPAddress,omitempty"`
+	PublicIPAddress *SubResource `json:"publicIPAddress,omitempty"`
```

Comment:
>SubResource [](http://example.com/codeflow?start=18&length=11)

can you please point me to the azure models.go? the one i am looking at, its not SubResource